app/src/main/java/com/moviefiend/torian/moviefiend/MovieAdapter.java

```diff
@@ -0,0 +1,55 @@
+package com.moviefiend.torian.moviefiend;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.moviefiend.torian.moviefiend.network.NowPlayingResponse;
+
+import java.util.ArrayList;
+
+
+public class MovieAdapter extends RecyclerView.Adapter<MovieViewHolder> {
+
+    private Context mContext;
+    private ArrayList<NowPlayingResponse.MovieInfo> mMovies;
+
+    public MovieAdapter(Context context) {
+        this.mContext = context;
+
+        mMovies = new ArrayList<>();
+    }
+
+    public void setMovies(ArrayList<NowPlayingResponse.MovieInfo> movies) {
```

Comment:
`List`
Here it's especially important since you don't to force the users of this method to pass a concrete implementation, they should be able to pass a `LinkedList` if they wish.